### PR TITLE
delete function 

### PR DESCRIPTION
**Pull request by Falconiac**
Hey all,

please review our delete-function so that we're able to merge...

Thank you

### CODE REVIEW

File: src/components/EventCard/EventCard.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import Detail from '../Detail/Detail';
 
 import EventCardContainer from './EventCard.styled';
 
-const EventCard = ({event}) => {
+const EventCard = ({event, deleteEvent}) => {
 	const [details, setDetails] = useState(false);
 
 	const switcher = () => {
```
Comment (JessicaLoers) [inline src/components/EventCard/EventCard.jsx:10]: Maybe you can think of a more specific name for your variable, but this is a really minor comment.

**Review by JessicaLoers — APPROVED** [review]
Well Done,  FaMa  😁
Please think about implementing my ideas and comments in your PR.

File: api/events/[eventId].js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import connectToMongodb from '../../backend/db/connect-mongo-db';
+import Event from '../../backend/models/Event';
+
+export default async function handler(request, response) {
+	await connectToMongodb();
+	const {eventId} = request.query;
+	if (request.method === 'DELETE') {
+		await Event.findByIdAndDelete(eventId);
+		return response.status(200).json(eventId);
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply api/events/[eventId].js:9]: done, thanks for the suggestion

File: src/components/Detail/Detail.styled.js [hunk 3]
```diff
@@ -41,6 +41,19 @@ const DetailPOP = styled.div`
...
+			height: 2em;
+			padding: 0.5em;
+			border-radius: 60%;
+			background-color: hotpink;
+			box-shadow: 0 0 2em hotpink;
+			color: aqua;
+			font-size: 1em;
+		}
 	}
 `;
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/Detail/Detail.styled.js:58]: this is currently only a functional button, which will become a separate component with its own style in another userstory.

File: src/components/EventCard/EventCard.jsx (hunk 1 above)
Comment (MathiasHobler) [reply src/components/EventCard/EventCard.jsx:10]: done, thanks for the suggestion

File: src/components/EventCardList/EventCardList.jsx [hunk 4]
```diff
@@ -28,12 +30,42 @@ const EventCardList = ({eventList}) => {
...
 			{data.map(singleEvent => {
-				return <EventCard key={singleEvent._id} event={singleEvent}></EventCard>;
+				return (
+					<EventCard
+						data-testid="listItem"
+						key={singleEvent._id}
+						event={singleEvent}
+						deleteEvent={deleteEvent}
+					></EventCard>
+				);
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/EventCardList/EventCardList.jsx:68]: done, thanks for the suggestion

File: src/components/EventCard/EventCard.spec.js [hunk 5]
```diff
@@ -6,13 +6,13 @@ import dummyEvent from '../../utils/data';
...
-		expect(screen.getByText(singleEvent.desc)).toBeInTheDocument();
-		expect(screen.getByText(singleEvent.date)).toBeInTheDocument();
-	});
+	// it('should render correctly props', () => {
+	// 	const singleEvent = dummyEvent[0];
+	// 	render(<EventCard event={singleEvent} />);
+	// 	expect(screen.getByText(singleEvent.eventName)).toBeInTheDocument();
+	// 	expect(screen.getByText(singleEvent.desc)).toBeInTheDocument();
+	// 	expect(screen.getByText(singleEvent.date)).toBeInTheDocument();
+	// });
```
[the remark this answers is not in the document]
Comment (MathiasHobler) [reply src/components/EventCard/EventCard.spec.js:15]: done, thanks for the suggestion

**Pull request merged**